### ISSUE TITLE
SO101 add degrees units for MolmoAct2

### ISSUE THREAD

**Issue by alfieroddan** [openvinotoolkit/physicalai#288]
## Problem

The pretrained MolmoAct2 SO-101 checkpoint uses degrees.

The native PhysicalAI Runtime SO-101 driver uses:

- Body joints: `[-100, 100]`
- Gripper: `[0, 100]`

MolmoAct2 currently converts the pretrained degree statistics using fixed SO-101 calibration ranges. Real calibration files have different ranges. This produces incorrect scaling during zero-shot inference.

https://github.com/open-edge-platform/physical-ai-studio/blob/main/library/src/physicalai/policies/molmoact2/constants.py

Fine-tuned models using PhysicalAI dataset statistics are not affected because they already train in Runtime’s normalized units.

LeRobot avoids this problem by using its legacy degree mode for zero-shot inference.

#### Option 1: Calibration-aware processors

Add Runtime processors that:

1. Read the SO-101 calibration.
2. Convert normalized observations to degrees before model inference.
3. Convert predicted degree actions back to normalized units afterward.

This keeps the robot driver in normalized mode and allows one model export to work with different calibrations.

The model manifest must clearly state when degree conversion is required.

#### Option 2: Add degree units to Runtime

Add `unit="degrees"` to the native Runtime SO-101 driver.

The driver would convert between ticks and degrees using the calibration midpoint and encoder resolution, matching LeRobot:

```text
degrees = (ticks - calibration midpoint) * 360 / 4095
```

Zero-shot MolmoAct2 could then use its original degree statistics directly. Fine-tuned PhysicalAI models would continue using `unit="normalized"`.

### Expected Result

Zero-shot MolmoAct2 inference should work with each robot’s calibration and should not depend on hardcoded calibration ranges.

**Comment by alfieroddan** [comment]
As mentioned in https://github.com/open-edge-platform/physical-ai-studio/issues/1204.

The library can handle the user's particular robot calibration, thus making minimal changes to runtime.
